solidity/contracts/bridge/library/FraudVerification.sol

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity 0.8.4;
+
+import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
+import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
+import {
+    CheckBitcoinSigs
+} from "@keep-network/bitcoin-spv-sol/contracts/CheckBitcoinSigs.sol";
+import "../Bridge.sol";
+
+library FraudVerification {
+    using BytesLib for bytes;
+    using BTCUtils for bytes;
+    using BTCUtils for uint32;
+
+    struct Parameters {
+        /// The amount of ETH the party challenging the wallet for fraud needs
+        /// to deposit.
+        uint256 fraudChallengeDepositAmount; //TODO: Initialize
+    }
+
+    struct FraudChallenge {
+        address challenger;
+        uint256 ethDepositAmount;
```

Comment:
Maybe just `depositAmount`?

Reply:
We talk a lot about deposits in the `Bridge`. Wouldn't it confuse somebody unfamiliar with the code what type of deposit we are talking about (bitcoin deposits, ether deposits)?

Reply:
That's a fair point but we are in the context of the `Fraud` library here. That reduces ambiguity. Going further, `eth` prefix does not add much context as well since we can potentially have multiple "eth deposits" in the future. Maybe we can name it `challengeDepositAmount` which seems to clarify all doubts. We also need to add a doc comment for it to clarify it is denominated in ETH.

Reply:
Done in 0907046b2b59de582bcd7dba8972a492b77404a4.